src/frontend/components/DashboardVideoLiveControlPane/widgets/DashboardVideoLiveWidgetLivePairing/index.tsx

```diff
@@ -0,0 +1,43 @@
+import { Box } from 'grommet';
+import React from 'react';
+import { defineMessages, useIntl } from 'react-intl';
+
+import { Video } from 'types/tracks';
+import { DashboardVideoLivePairing } from 'components/DashboardVideoLivePairing';
+import { DashboardVideoLiveWidgetTemplate } from 'components/DashboardVideoLiveControlPane/widgets/DashboardVideoLiveWidgetTemplate';
+
+const messages = defineMessages({
+  externalSourcesTitle: {
+    defaultMessage: 'External broadcast sources',
+    description: 'Title used in the external broadcast dashboard widget',
+    id: 'components.DashboardVideoLiveControlPane.widgets.DashboardVideoLiveWidgetLiePairing.title',
+  },
+  exteralSourcesInfo: {
+    defaultMessage:
+      'The appairing button allows you to appair an external device and connect you to the current jitsi room.',
+    description:
+      'Helper explaining what contains the external sources widget and how to use the appairing button',
+    id: 'components.DashboardVideoLiveControlPane.widgets.DashboardVideoLiveWidgetLiePairing.info',
+  },
+});
+
+interface DashboardVideoLiveWidgetLivePairingProps {
+  video: Video;
+}
+
+export const DashboardVideoLiveWidgetLivePairing = ({
+  video,
+}: DashboardVideoLiveWidgetLivePairingProps) => {
+  const intl = useIntl();
+  return (
+    <DashboardVideoLiveWidgetTemplate
+      title={intl.formatMessage(messages.externalSourcesTitle)}
+      infoText={intl.formatMessage(messages.exteralSourcesInfo)}
+      initialOpenValue={false}
+    >
+      <Box direction={'row'} justify={'center'} margin={'small'}>
```

Comment:
```suggestion
      <Box
        direction={'row'}
        justify={'center'}
        margin={{ horizontal: 'small' }}
      >
```
The widget's template already has a padding, I don't think its necessary to add some vertical margin.